doc/source/ray-air/examples/batch_forecasting.ipynb

```diff
@@ -1168,7 +1168,7 @@
     "- We will restore a Prophet or ARIMA model directly from checkpoint, and demonstrate it can be used for prediction.\n",
     "\n",
     "```{tip}\n",
-    "[Ray AIR Predictors](air-predictors) make batch inference easy since they have internal logic to parallelize the inference.\n",
+    "Ray AIR Predictors make batch inference easy since they have internal logic to parallelize the inference.\n",
```

Comment:
```suggestion
    "Ray Predictors make batch inference easy since they have internal logic to parallelize the inference.\n",
```

Reply:
Let's defer the "AIR" removals - "Ray Predictors" make it sound like a first class concept, which it's not. The AIR removal hasn't been fully done, yet - IMO this should be part of that. here we just remove the references